### ISSUE TITLE
Windows Navigation Animation jerky

### ISSUE THREAD

**Issue by sbloomztf** [dotnet/maui#26432]
### Description

On 9.0.12, animation when navigating between views is no longer a smooth transition.   The background flashes to a default windows background color, replacing the app's background color / image. 

### Steps to Reproduce

Create a new project with views that have a background color/image set.  Navigate to another view or shell tab to see the issue.

This issue is not present on 9.0.10, and is on 9.0.12

### Version with bug

9.0.12 SR1.2

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

9.0.10 SR1

### Affected platforms

Windows

### Affected platform versions

Windows

### Did you find any workaround?

Revert back to 9.0.10

### Relevant log output

```shell

```

**Comment by jfversluis** [comment]
I think we will need a bit more information, this is a bit thin. Do you have a screen recording of before and after? Is this really visible with just a File > New MAUI project? Does this reproduce with a WinUI app without MAUI involved? Maybe its the WinUI version bump that is causing this.

Thanks!

cc: @Foda does this mean anything to you?

**Comment by sbloomztf** [comment]
Here's an example repo.

https://github.com/sbloomztf/MauiWindowsNavsAJerk

switch between 9.0.10 and 9.0.12 to see the differences in navigation animation

**Comment by PureWeen** [comment]
@sbloomztf I've tried your sample on 9.0.12 and 9.0.10 and they both work the same for me

**Comment by jeremy-visionaid** [comment]
I also saw a regression in navigation on Windows in 9.0.12, I'm not sure if they might be related. In #26466 it ends up affecting the enabled/disabled state of a button, which is a fair bit worse than the animation just being janky 😬

**Comment by Redth** [comment]
So I think I see the difference here, but it's a bit subtle.

## 9.0.10
https://github.com/user-attachments/assets/6bf5951f-f24e-47ba-8738-028e910162b7

## 9.0.12
https://github.com/user-attachments/assets/0b940201-3ee4-43bf-be01-6afd710e076f

Notice that in 9.0.12, there's a brief 'flash' before the slide animation?

**Comment by sbloomztf** [comment]
> So I think I see the difference here, but it's a bit subtle.
> Notice that in 9.0.12, there's a brief 'flash' before the slide animation?

Yes, this is the flicker I am reporting.   The issue may be even more obvious on slower machines.

**Comment by Ying-6** [comment]
This issue has been verified using Visual Studio 17.13.0 Preview 1.0 (9.0.12). Can repro on Windows and Android Platforms. And 9.0.10 & 8.0.100 works fine.
![Image](https://github.com/user-attachments/assets/ca57d830-00cd-43f7-9da8-f6e65c477b5e)

**Comment by RuddyOne** [comment]
I also have this issue on my Windows app and that's .NET 8. I'm pretty sure this was an existing issue. I have never not seen this behaviour. I thought there was an issue open for this but am failing to find it. 

I am just updating to .NET 9 to see if this issue resolves or is still there. But this is 100% an existing bug.
